Pack-Smart/pack-smart-fe
# Refactor/redux connect

## What's this PR do?
- Refactors how we wrote our connects
- ALSO! Somehow our deleteSingleItem stuff in VerifyDeletionModal got deleted during a merge. I put it all back! :) 

## Where should the reviewer start?
- Check all the components with `connect()`
- See VerifyDeletionModal ... the code should be back in there!

## How should this be manually tested?
- Try to delete an item from a packing list. 
- Leave the page and come back and those items should NOT be there!

## Any background context you want to provide?
- I did not try to fix our naming conventions. That seems like a much bigger refactor than we have time for before Code Freeze!

## What are the relevant tickets?
- Closes #105 


## Did you update the README?
- No
